### PR TITLE
sps: Fix importing large number of sessions

### PR DESCRIPTION
**Pull request by davedusa**
Changes:
- sps: Fix importing large number of sessions
- sps: Add separate troubleshooting guide
- sps: Fix not-working links in README
- sps: Fix adv. search responses to match reality
- sps: Fix typo

### CODE REVIEW

File: SafeguardSessions/TROUBLESHOOTING.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+This error wraps unexpected errors raised by the Power BI Connector. Reproduce the error with trace logging enabled, and create a technical case, as described in the [Troubleshooting] section. Attach the mashup trace logs to the issue.
+
+<!-- Links -->
+
+[Troubleshooting]: #troubleshooting
+
+[Installation and usage]: README.md#installation-and-usage
+[Upgrading the Power BI Connector]: README.md#upgrading-the-power-bi-connector
+
+[Technical documents for One Identity Safeguard for Priviledged Sessions]: https://support.oneidentity.com/one-identity-safeguard-for-privileged-sessions/technical-documents
```
Comment (asulyi) [inline SafeguardSessions/TROUBLESHOOTING.md:87]:
```suggestion
[Technical documents for One Identity Safeguard for Privileged Sessions]: https://support.oneidentity.com/one-identity-safeguard-for-privileged-sessions/technical-documents
```
also in
https://github.com/OneIdentity/SafeguardPowerBI/blob/c1ab3d3a2bb65d9719e4e77e8bc881c4f9184ac7/SafeguardSessions/TROUBLESHOOTING.md?plain=1#L34
and
https://github.com/OneIdentity/SafeguardPowerBI/blob/c1ab3d3a2bb65d9719e4e77e8bc881c4f9184ac7/SafeguardSessions/TROUBLESHOOTING.md?plain=1#L58

**Review by asulyi — CHANGES_REQUESTED** [review]
I've only found another typo.

File: SafeguardSessions/TROUBLESHOOTING.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+## Troubleshooting
+
+If an error occurs during the data fetching process, you can check the **Info** table to see a descriptive message about the error type. In this case, the **Sessions** table will contain the exact error record that you can inspect if you go to **Home > Transform data** and select the **Sessions** table.
+
```
Comment (marcellmeszaros) [inline SafeguardSessions/TROUBLESHOOTING.md:4]: This case above, will only occur when loading data and the error occurs on the first data request. If you run into an error in the middle of a data load, then you will get a pop up in that case. And if this error is not temporary, then on reload we will get the Info and Sessions table that contains the " extracted " error.

File: SafeguardSessions/TROUBLESHOOTING.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+## Troubleshooting
+
+If an error occurs during the data fetching process, you can check the **Info** table to see a descriptive message about the error type. In this case, the **Sessions** table will contain the exact error record that you can inspect if you go to **Home > Transform data** and select the **Sessions** table.
+
+If you have already and successfully imported data from SPS and you run into an error when refreshing data so that the data is in the **Info** and **Sessions** table kept intact, you can find more details about the cause of the error in this guide.
+
```
Comment (marcellmeszaros) [inline SafeguardSessions/TROUBLESHOOTING.md:6]: Here too, the loaded data will only be retained on refresh if an error is not returned on the first data query.

**Review by marcellmeszaros — CHANGES_REQUESTED** [review]
I think the Troubleshooting.md in commit 2 (sps: Add separate troubleshooting guide) should be clarified a bit, how pop-up and non-pop errors can occur.

File: SafeguardSessions/TROUBLESHOOTING.md (hunk 3 above)
Comment (davedusa) [reply SafeguardSessions/TROUBLESHOOTING.md:6]: See the [previous answer](https://github.com/OneIdentity/SafeguardPowerBI/pull/58#discussion_r1330579865).

File: SafeguardSessions/TROUBLESHOOTING.md (hunk 1 above)
Comment (davedusa) [reply SafeguardSessions/TROUBLESHOOTING.md]: Thanks! Done.

File: SafeguardSessions/CHANGELOG.md
Comment (szszeker) [inline SafeguardSessions/CHANGELOG.md:1]: Small typo in the commit message, I don't know if the other noticed it or not.
if there were _soo_ many sessions (~1.5 million) - > if there were _too_ many sessions (~1.5 million)

**Review by szszeker — COMMENTED** [review]
Everything else seems fine.

**Review by asulyi — APPROVED** [review]

**Review by marcellmeszaros — APPROVED** [review]

**Pull request merged**